### PR TITLE
BHBC-2306: Hide Submit functionality

### PR DESCRIPTION
**Pull request by AlfredRosenthal**
# Overview

## Links to Jira tickets

[BHBC-2306](https://quartech.atlassian.net/browse/BHBC-2306?atlOrigin=eyJpIjoiZjM3NjAwOTc0ZTY0NGUxMGFmNTNiYTEwMDM1YTk2ZWMiLCJwIjoiaiJ9)

## Description of relevant changes

- Added System Role Guard to various status columns and submission buttons
- Added `TODO: PRODUCTION_BANDAGE` to the pages to mark what changes to be removed later

## Testing Steps
- Login as a Data/ System admin
- Create a project and survey
- Add documents/ observations to the Survey and see submit buttons and status on documents
- Add documents to project and see submission status and submit button
- Modify current permissions to be a Creator (this will throw an error but can be ignored for now. Issue tracked in [BHBC-2309](https://quartech.atlassian.net/browse/BHBC-2309?atlOrigin=eyJpIjoiNTZiNDZmOWM5ZTRkNDU2M2IxNDFjNzY5ZThmMjEyMmIiLCJwIjoiaiJ9)
- Step through newly created project/ survey, all status and submission information will now be hidden

## PR Checklist

A list of items that are good to consider when making any changes.

_Note: this list is not exhaustive, and not all items are always applicable._

### Code

- [ ] New files/classes/functions have appropriately descriptive names and comment blocks to describe their use/behaviour
- [ ] I have avoided duplicating code when possible, moving re-usable pieces into functions
- [ ] I have avoided hard-coding values where possible and moved any re-usable constants to a constants file
- [ ] My code is as flat as possible (avoids deeply nested if/else blocks, promise chains, etc)
- [ ] My code changes account for null/undefined values and handle errors appropriately
- [ ] My code uses types/interfaces to help describe values/parameters/etc, help ensure type safety, and improve readability

### Style

- [ ] My code follows the established style conventions
- [ ] My code uses native material-ui components/icons/conventions when possible

### Documentation

- [ ] I have commented my code sufficiently, such that an unfamiliar developer could understand my code
- [ ] I have added/updated README's and related documentation, as needed

### Tests

- [ ] I have added/updated unit tests for any code I've added/updated
- [ ] I have added/updated the Postman requests/tests to account for any API endpoints I've added/updated

### Linting/Formatting

- [ ] I have run the linter and fixed any issues, as needed  
       _See the `lint` commands in package.json_
- [ ] I have run the formatter and fixed any issues, as needed  
       _See the `format` commands in package.json_

### SonarCloud

- [ ] I have addressed all SonarCloud Bugs, Vulnerabilities, Security Hotspots, and Code Smells

### CODE REVIEW

File: app/src/components/attachments/list/AttachmentsList.tsx [hunk 1]
```diff
@@ -58,8 +62,11 @@ const AttachmentsList = <T extends IGetProjectAttachment | IGetSurveyAttachment>
         <TableHead>
           <TableRow>
             <TableCell>Name</TableCell>
+            <TableCell width="80"></TableCell>
```
Comment (curtisupshall) [inline app/src/components/attachments/list/AttachmentsList.tsx]: Do we need this empty cell?

Comment (AlfredRosenthal) [reply app/src/components/attachments/list/AttachmentsList.tsx]: nope

**Review by curtisupshall — APPROVED** [review]
Tested and works as expected!

**Comment by curtisupshall** [comment]
If this PR started as a draft, you need to make a commit with "ignore-skip" in the message:

```
git commit --allow-empty -m "ignore-skip"
```

**Review by KjartanE — APPROVED** [review]
😹

**Pull request merged**